tests/scheduler/genetic/converter_test.py

```diff
@@ -6,10 +6,10 @@
 from sampo.utilities.validation import validate_schedule
 
 
-def test_convert_schedule_to_chromosome(setup_toolbox):
+def test_convert_schedule_to_chromosome(setup_toolbox, setup_landscape_many_holders):
```

Comment:
`setup_landscape_many_holders` should come from `setup_scheduler_parameters`, and from `setup_toolbox` too.
Here you just duplicating tests.